tests/gw/test_gw_likelihood.py

```diff
@@ -0,0 +1,233 @@
+import bilby
+import numpy as np
+import pandas as pd
+import pytest
+from bilby.gw import GravitationalWaveTransient
+from bilby.gw import WaveformGenerator as BilbyWaveformGenerator
+from bilby.gw.detector import InterferometerList
+
+from dingo.gw.domains import build_domain
+from dingo.gw.injection import Injection
+from dingo.gw.likelihood import StationaryGaussianGWLikelihood
+from dingo.gw.prior import (
+    build_prior_with_defaults,
+    default_intrinsic_dict,
+    default_extrinsic_dict,
+)
+from dingo.gw.waveform_generator import NewInterfaceWaveformGenerator
+
+
+@pytest.fixture(params=["IMRPhenomXPHM", "SEOBNRv4PHM", "SEOBNRv5PHM"])
+def approximant(request):
+    return request.param
+
+
+@pytest.fixture
+def bbh_parameters():
+    return {
+        "mass_1": 60.29442201204798,
+        "mass_2": 25.460299253933126,
+        "phase": 2.346269257440926,
+        "a_1": 0.07104636316747037,
+        "a_2": 0.7853578509086726,
+        "tilt_1": 1.8173336549500292,
+        "tilt_2": 0.4380213394743055,
+        "phi_12": 5.892609139936818,
+        "phi_jl": 1.6975651971466297,
+        "theta_jn": 1.0724395559873239,
+        "luminosity_distance": 1000.0,
+        "geocent_time": 0.0,
+        "ra": 1.0,
+        "dec": 2.0,
+        "psi": 2.5,
+    }
+
+
+@pytest.fixture  # (params=["IMRPhenomXPHM", "SEOBNRv4PHM", "SEOBNRv5PHM"])
+def injection(approximant):
```

Comment:
Consider renaming the function to something like `prepare_injection()` to make the later usage `injection.injection` clearer.